pyproject.toml

```diff
@@ -1,45 +1,43 @@
-[tool.poetry]
+[project]
 name = "pyfixest"
-version = "0.24.2"
-
-description = "Fast high dimensional fixed effect estimation following syntax of the fixest R package. Supports OLS, IV and Poisson regression and a range of inference procedures (HC1-3, CRV1 & CRV3, wild bootstrap, randomization inference, simultaneous CIs, Romano-Wolf's multiple testing correction). Additionally, supports (some of) the regression based new Difference-in-Differences Estimators (Did2s, Linear Projections)."
-authors = ["Alexander Fischer <alexander-fischer1801@t-online.de>", "Styfen Schär"]
-license = "MIT"
-readme = "README.md"
-homepage = "https://py-econometrics.github.io/pyfixest/"
-repository = "https://github.com/py-econometrics/pyfixest"
-
-[tool.poetry.dependencies]
-python = ">=3.9,<4.0"
-pandas = ">=1.1.0"
-numpy = ">=1.19.0"
-scipy = ">=1.6"
-formulaic = ">=1.0.0"
-lets-plot = ">=4.0.1"
-seaborn = ">=0.13.2"
-numba = ">=0.58.0"
-tabulate = ">=0.9.0"
-polars = ">=0.20.1"
-tqdm = "^4.66.4"
-great-tables = ">=0.10.0"
-
-[tool.poetry.group.dev.dependencies]
-pytest=">=7.0.0"
-rpy2 = "3.5.15"
-pytest-cov = "^4.1.0"
-pytest-xdist = ">=3.5.0"
-pyhdfe = ">=0.2.0"
-pyarrow = ">=14.0.2"
-wildboottest = ">=0.2.0"
-pre-commit = "^3.6.0"
-doubleml = "^0.7.1"
-marginaleffects = "^0.0.10"
-
-[tool.poetry.group.docs.dependencies]
-quartodoc = ">=0.7.2"
-jupyterlab = "^4.0.12"
-jupytext = "^1.16.1"
-watermark = "^2.4.3"
+description = "Fast high dimensional fixed effect estimation following syntax of the fixest R package."
+requires-python = ">= 3.9"
+version = "0.24.3"
+authors = [{name = "Alexander Fischer", email = "alexander-fischer1801@t-online.de"}, {name = "Styfen Schär"}]
+dependencies = ["lets-plot>=4.0.0", "scipy>=1.6", "formulaic>=1.0.0", "pandas>=1.1.0", "numba>=0.58.0", "seaborn>=0.13.2", "tabulate>=0.9.0", "polars>=0.20.1", "tqdm==4.66.4", "great-tables>=0.10.0", "numpy>=1.19.0"]
```

Comment:
can we make this list have the form (one package per line)

```python
["lets-plot>=4.0.0",
...,
 "numpy>=1.19.0", 
]
so that is easier to read :) (same with other dependencies below)

Reply:
fixed now =)